### ISSUE TITLE
VALARM event properties not carried over

### ISSUE THREAD

**Issue by Nycrea** [niccokunzmann/python-recurring-ical-events#6]
When parsing a calendar with recurring_ical_events, the resulting events differ from the original in that they do not contain the VALARM attribute. 

Example event input
```
BEGIN:VCALENDAR
BEGIN:VEVENT
SUMMARY:redacted
DTSTART;TZID=Europe/Berlin:20190527T140000
DTEND;TZID=Europe/Berlin:20190527T163000
DTSTAMP:20190510T070457Z
UID:00000000-0000-0000-0000-000000000000
SEQUENCE:4
ATTENDEE;CN=redacted;PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CN="redacted";PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CN="redacted";PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CN="redacted";PARTSTAT=ACCEPTED;ROLE=REQ-PARTICIPANT:mailto:redacted@example.com
ATTENDEE;CUTYPE=RESOURCE;PARTSTAT=ACCEPTED;ROLE=NON-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CUTYPE=RESOURCE;PARTSTAT=ACCEPTED;ROLE=NON-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
CLASS:PUBLIC
DESCRIPTION:redacted
LAST-MODIFIED:20190510T070457Z
LOCATION:redacted@example.com
ORGANIZER;CN=redacted;SENT-BY="mailto:redacted@example.com":mailto:redacted@example.com
STATUS:CONFIRMED
TRANSP:OPAQUE
X-MICROSOFT-CDO-INTENDEDSTATUS:BUSY
X-MS-OLK-SENDER:mailto:redacted@example.com
BEGIN:VALARM
ACTION:DISPLAY
DESCRIPTION:Reminder
TRIGGER;RELATED=START:-PT10M
END:VALARM
END:VEVENT
END:VCALENDAR
```
Output:
```
BEGIN:VCALENDAR
BEGIN:VEVENT
SUMMARY:redacted
DTSTART;TZID=Europe/Berlin:20190527T140000
DTEND;TZID=Europe/Berlin:20190527T163000
DTSTAMP:20190510T070457Z
UID:00000000-0000-0000-0000-000000000000
SEQUENCE:4
ATTENDEE;CN=redacted;PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CN="redacted";PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CN="redacted";PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CN="redacted";PARTSTAT=ACCEPTED;ROLE=REQ-PARTICIPANT:mailto:redacted@example.com
ATTENDEE;CUTYPE=RESOURCE;PARTSTAT=ACCEPTED;ROLE=NON-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
ATTENDEE;CUTYPE=RESOURCE;PARTSTAT=ACCEPTED;ROLE=NON-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
CLASS:PUBLIC
DESCRIPTION:redacted
LAST-MODIFIED:20190510T070457Z
LOCATION:redacted@example.com
ORGANIZER;CN=redacted;SENT-BY="mailto:redacted@example.com":mailto:redacted@example.com
STATUS:CONFIRMED
TRANSP:OPAQUE
X-MICROSOFT-CDO-INTENDEDSTATUS:BUSY
X-MS-OLK-SENDER:mailto:redacted@example.com
END:VEVENT
END:VCALENDAR
```

**Comment by Nycrea** [comment]
This seems to be caused by `event.copy()` which does not copy this property. I am planning to fix this by replacing the dates and recurrence stuff in the original event using your code instead of creating a new event with `event.copy()`

**Comment by Nycrea** [comment]
I just noticed that this property is not part of the event, but instead of the calendar, and hence it is my own fault for not copying it.

**Comment by niccokunzmann** [comment]
However: If this is caused by event.copy and this package uses event.copy and you expect it to work otherwise, you can leave this open, open an issue in the icalendar package, reference this and see where this should be fixed. Maybe it is default in the icalendar package and ok and unexpected in this one.
Let me know what you think or if you think something should be changed here.

**Comment by niccokunzmann** [comment]
If you have a look at the documentation, I would say that what you found out could at least be documented for the users: https://github.com/niccokunzmann/python-recurring-ical-events#events-as-list

I wonder what is specified in icalendar. Research:
- https://github.com/collective/icalendar/search?utf8=%E2%9C%93&q=copy&type=
- https://github.com/collective/icalendar/blob/194338629370ada3a2978c42bbe85529f8012d3d/src/icalendar/tests/test_unit_caselessdict.py#L68
- https://github.com/collective/icalendar/blob/f340b761aa3a4a6f6b456833cc29ef1b05cd9c14/src/icalendar/cal.py#L84

Copying seems to not include subcomponents. I would like to have this documented.

**Comment by niccokunzmann** [comment]
This fix is included in v0.1.15b. It is expected and useful to handle it like this.
